codecov.yaml

```diff
@@ -0,0 +1,3 @@
+ignore:
+  #- "path/to/folder"   ignore folders and all its contents
+  - "./ThirdParty"
```

Comment:
Ignoring the coverage for [sqlite3.c](https://github.com/mozilla-mobile/firefox-ios/blob/master/ThirdParty/sqlcipher/sqlite3.c) file is not shown which was annoying being showed in all pr as the coverage decreases there